### ISSUE TITLE
The python error message isn't returned when running over IPC

### ISSUE THREAD

**Issue by tepu** [KxSystems/embedPy#19]
If the evaluation errors when running over IPC, the error message is always the same
```
q)server ".p.eval \"[1,2,3][10]\""
'p.c:68 runs pyerr
  [0]  server ".p.eval \"[1,2,3][10]\""
       ^
q)server ".p.eval \"1 + myUndefinedVar\""
'p.c:68 runs pyerr
  [0]  server ".p.eval \"1 + myUndefinedVar\""
       ^
q)server ".p.eval \"[1,2,3)\""
'p.c:68 runs pyerr
  [0]  server ".p.eval \"[1,2,3)\""
```
These messages are printed in the server process, but aren't accessible from the client process
```
IndexError: list index out of range
NameError: name 'myUndefinedVar' is not defined
SyntaxError: unexpected EOF while parsing
```

**Comment by abalkin** [comment]
In PyQ we translate Python errors to `'ErrrorType`:

```q
q)p)1+`
'SyntaxError
q)p)1+""
'TypeError
```

It should not be hard to replicate the same logic in embedPy.

**Comment by tepu** [comment]
It would be helpful if it could return the whole error message
```NameError: name 'myUndefinedVar' is not defined```
instead of just
```NameError```

**Comment by jfakx** [comment]
this does need refinement.  currently, a python error is printed and a signal raised.
maybe we could keep the error short, but provide a function to get extended error info `xrr`

**Comment by jhanna-kx** [comment]
Hi
This should be fixed now, since release [1.1](../releases/1.1) you'll get this:

```q
q)server"p)1 + myUndefinedVar"
'e: name 'myUndefinedVar' is not defined
  [0]  server"p)1 + myUndefinedVar"
       ^
```

**Comment by jhanna-kx** [comment]
closing, feel free to reopen if new behaviour doesn't work
